.editorconfig

```diff
@@ -0,0 +1,12 @@
+# http://editorconfig.org
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[{*.js,*.json,*.yml}]
+indent_size = 2
+indent_style = space
```

Comment:
Why not put these 2 in just `*`?

Reply:
I believe we will not add other languages except ts typings in most of our packages so * is very general mask. Let's add `*.ts` and maybe `*.h,*.cc,*.c`

Reply:
I'd like to have common `.editorconfig` for almost all modules in Metarhia